src/Common/File.h

```diff
@@ -20,12 +20,14 @@
 
 #pragma once
 
-#include <filesystem>
+#include <experimental/filesystem>
```

Comment:
This could be wrapped inside `__has_include` check verifying presence of non-experimental FS, but this should work as-is in most toolchains. We use experimental/boost FS like so in OpenLoco: https://github.com/OpenLoco/OpenLoco/blob/master/src/openloco/platform/platform.h